draft-dnsop-deleg-transport.md

```diff
@@ -196,6 +202,9 @@ alpn=doq tlsa="..." mandatory=alpn
 ;; services.  All resolvers can access it, but only resolvers with
 ;; DANE support will use DoQ.
 alpn=doq tlsa="..."
+[... how does this signal that it also offers do53? Is the though that it is
+  because only this draft specifies alpn and the core draft doesn't (currently)
+  have a method to explicitly signal do53?  Maybe it should... ]
```

Comment:
This server supports Do53 because it is not explicitly disclaimed by `mandatory=alpn`, as in the last example here.

Reply:
Okay, right, see above.  I think we should require signalling do53 to be explicit.

Reply:
```suggestion
[ If we move to requiring an explicit opt-in for do53 then these examples will have to change. ]
```